bundles/CoreBundle/Migrations/Version20211103055110.php

```diff
@@ -0,0 +1,52 @@
+<?php
```

Comment:
Please add strict type declaration
```suggestion
<?php

declare(strict_types=1);
```